base/linalg/symmetric.jl

```diff
@@ -218,6 +248,10 @@ A_mul_B!{T<:BlasComplex,S<:StridedMatrix}(C::StridedMatrix{T}, A::StridedMatrix{
 *(A::HermOrSym, B::HermOrSym) = full(A)*full(B)
 *(A::StridedMatrix, B::HermOrSym) = A*full(B)
 
+for T in (:Symmetric, :Hermitian), op in (:+, :-, :*, :/)
+    @eval ($op)(A::$T, x::Number) = ($T)(($op)(A.data, x), Symbol(A.uplo))
```

Comment:
The `Hermitian` operations should probably be restricted to operations with `x::Real`.